aspnetcore/fundamentals/http-requests.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Initiate HTTP requests
+author: stevejgordon
+description: Learn about using the HttpClientFactory to managed logic HttpClient instances.
+manager: wpickett
+ms.author: riande
+ms.date: 02/15/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/http-requests
+---
+# Initiate HTTP requests
+
+By [Glenn Condron](https://github.com/glennc), [Ryan Nowak](https://github.com/rynowak) and [Steve Gordon](https://github.com/stevejgordon) 
+
+A `HttpClientFactory` can be registered and used to configure and consume `HttpClient` instances in an app. It provides several benefits:
+
+- Provides a central location for naming and configuring logical `HttpClient` instances. For example, a "github" client can be registered and pre-configured to access GitHub while a default client can be registered for other purposes.
+- Codifies the concept of outgoing middleware via delegating handlers in `HttpClient` and provides extensions for Polly-based middleware to take advantage of that. 
+- Manages the pooling and lifetime of underlying `HttpClientMessageHandler` instances to avoid common DNS problems that can be hit when managing `HttpClient` lifetimes manually.
+
+## Consumption patterns
+
+There are several ways that `HttpClientFactory` can be used in an app. None of them are strictly superior to another, it really depends on the constraints on the app.
+
+## Basic usage
+
+The `HttpClientFactory` can be used directly in code to access `HttpClient` instances. First, the services must be registered with the ServiceProvider.
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddHttpClient();
+    services.AddMvc();
+}
+```
+
+Once registered, code can accept a `IHttpClientFactory` anywhere services can be injected by the DI framework. The `IHttpClientFactory` can then be used to create a `HttpClient` instance.
```

Comment:
~~can be injected by the DI framework.~~
can be injected with dependency injection (DI).